### PR TITLE
Add some type hinting

### PR DESCRIPTION
**Pull request by Shrews**
We now have a `mypy` linting job thanks to PR #1254, so let's improve code health by adding some type hints.

### CODE REVIEW

File: src/ansible_runner/cleanup.py [hunk 1]
```diff
@@ -143,7 +151,7 @@ def cleanup_dirs(pattern, exclude_strings=(), grace_period=GRACE_PERIOD_DEFAULT)
     return ct
 
 
-def cleanup_images(images, runtime='podman'):
+def cleanup_images(images: list, runtime: str) -> int:
```
Comment (Akasurde) [inline src/ansible_runner/cleanup.py:154]:
```suggestion
def cleanup_images(images: list, runtime: str='podman') -> int:
```
Missing default value?

**Review by Akasurde — APPROVED** [review]
Rest LGTM

File: src/ansible_runner/cleanup.py (hunk 1 above)
Comment (Shrews) [reply src/ansible_runner/cleanup.py:154]: Nope. I changed the calls to `cleanup_images()` to make sure this value is never `None` by using the default runtime at that point.

Comment (Akasurde) [reply src/ansible_runner/cleanup.py:154]: Ok. Got it.

**Pull request merged**